utils/validate-facility-ids/index.ts

```diff
@@ -0,0 +1,37 @@
+import { difference } from 'lodash';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { mongoDbClient } from './database-client';
+import { getUniqueFeeRecordFacilityIds, getUniqueTfmFacilitiesUkefFacilityIds } from './helpers';
+
+const run = async (): Promise<void> => {
+  const dataSource = await SqlDbDataSource.initialize();
+  try {
+    const tfmFacilitiesUkefFacilityIds = await getUniqueTfmFacilitiesUkefFacilityIds();
+    const feeRecordFacilityIds = await getUniqueFeeRecordFacilityIds(dataSource);
+
+    const tfmFacilitiesFacilityIdsNotInFeeRecord = difference(tfmFacilitiesUkefFacilityIds, feeRecordFacilityIds);
+    const feeRecordFacilityIdsNotInTfmFacilities = difference(feeRecordFacilityIds, tfmFacilitiesUkefFacilityIds);
+
+    if (tfmFacilitiesFacilityIdsNotInFeeRecord.length === 0 && feeRecordFacilityIdsNotInTfmFacilities.length === 0) {
+      console.info('All fee record facility ids exist in the tfm facilities collection');
+      return;
+    }
+
+    tfmFacilitiesFacilityIdsNotInFeeRecord.forEach((facilityId) => {
+      console.info(`Facility id '${facilityId}' exists in the "tfm-facilities" collection but not in the FeeRecord table`);
+    });
```

Comment:
Is it not expected that there will be lots of facility ids without fee records and we don't care about that? e.g. banks which use the portal but not for utilisation reporting? I think we only care about fee records without a corresponding facility